inffast.c

```diff
@@ -249,7 +249,7 @@ void Z_INTERNAL zng_inflate_fast(PREFIX3(stream) *strm, unsigned long start) {
                         from += wsize - op;
                         if (op < len) {         /* some from end of window */
                             len -= op;
-                            out = functable.chunkcopy_safe(out, from, op, safe);
+                            out = functable.chunkcopy_safe(out, from, op, (uint32_t)(safe - out) + 1);
```

Comment:
Cast should be protected with extra `()` to avoid override of implicit cast with different signedness. Return value of pointer subtraction is `int32_t` on 32-bit platforms and `int64_t` on 64-bit platforms.

Reply:
Fixed thanks.